multifile/paster/vcf/__main__.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+
+from multifile.paster.vcf.paster import VCFPaster
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Merge VCF horizontally.")
+    parser.add_argument('input_paths', type=str, nargs='+', help="paths to files to merge")
```

Comment:
We should have an option to take a file that has a list of VCF filenames. We might use this on 50k files. Much easier for the user to make a file of filenames than force using xargs/whatever because of character/argument limits from the OS.

Reply:
@martinghunt :+1: Added. I refactored the code to be more compact, too. Could you help me check once you have time? Thank you.